services/mail/Makefile

```diff
@@ -0,0 +1,29 @@
+.DEFAULT_GOAL := help
+
+
+# Internal VARIABLES ------------------------------------------------
+# STACK_NAME defaults to name of the current directory. Should not to be changed if you follow GitOps operating procedures.
+STACK_NAME = $(notdir $(shell pwd))
+SWARM_HOSTS = $(shell docker node ls --format={{.Hostname}} 2>/dev/null)
+TEMP_COMPOSE=.stack.${STACK_NAME}.yaml
+
+
+# External VARIABLES
+include .env
+
+# TARGETS --------------------------------------------------
+include $(realpath $(CURDIR)/../../scripts/common.mk)
```

Comment:
MINOR: should addopt a naming convention regarding common makefiles.  See ``scripts/common.mk`` and ``services/common.Makefile``. The latter, I assume is only for services, but what about the former??

Reply:
Renamed it in common.makefile

Reply:
The script one add some general things like "info, help, clean" but need a little bit of refactoring, because somes recipe like clean are present in the services one.